public/less/admin/admin.less

```diff
@@ -213,24 +216,36 @@
 		}
 	}
 
-	.navbar {
-		padding: 0 5px;
-
-		.nav-home a, .nav-home a:hover {
-			width: 30px;
-			height: 30px;
-			padding: 5px;
-			text-align: center;
-			margin-top: 10px;
-			background: #DDD;
-
-			i {
-				color: black;
-				font-size: 17px;
-			}
-		}
-
-	}
+    .navbar-static-top, .navbar-fixed-top {
+        box-shadow: 0px -3px 12px rgba(0, 0, 0, 0.5);
+    }
+
+    .navbar-header > .navbar-toggle {
+        margin-right: 8px;
+    }
+
+    .navbar-nav {
+        margin-top: 0;
+        margin-bottom: 0;
+
+        >li {
+            >a {
+                padding-top: 15px;
+                padding-bottom: 15px;
+            }
+            >a:hover, >a:focus {
+                color:#333;
+                background-color:#b2cdf0
```

Comment:
For maximum portability, use:

* `@gray-dark` here instead of `#333`
* `@gray-light` instead of `#b2cdf0`

I personally find `#b2cdf0` a tad bright, but that is purely a personal choice.